src/Extensions/ExtHostClient.re

```diff
@@ -66,14 +67,29 @@ let start =
       let id = Protocol.PackedString.parse(v);
       onDidActivateExtension(id);
       Ok(None);
+    | (
+        "MainThreadExtensionService",
+        "$onExtensionActivationFailed",
+        [v, ..._],
+      ) =>
+      let id = Protocol.PackedString.parse(v);
+      onExtensionActivationFailed(id);
+      Ok(None);
     | ("MainThreadCommands", "$registerCommand", [`String(v), ..._]) =>
       onRegisterCommand(v);
       Ok(None);
     | ("MainThreadStatusBar", "$setEntry", args) =>
       In.StatusBar.parseSetEntry(args) |> apply(onStatusBarSetEntry);
       Ok(None);
     | (s, m, _a) =>
-      Log.error("Unhandled message - " ++ s ++ ":" ++ m ++ " | ");
+      Log.error(
+        "Unhandled message - "
+        ++ s
+        ++ ":"
+        ++ m
+        ++ " | "
+        ++ Yojson.Safe.to_string(`List(_a)),
+      );
```

Comment:
Sometimes I think `refmt` might be just a little bit silly (but oh my is it better than it was a bit over a year ago). I think this would look much more readable with `Printf.sprintf`.

Reply:
Definitely looks better with `Printf.sprintf` - thank you!